bucket/paint.net-plugin-boltbait.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "5.1",
+    "description": "Collection of paint.net plugins by BoltBait",
+    "homepage": "https://boltbait.com/pdn/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://boltbait.com/pdn/"
+    },
+    "depends": "paint.net",
+    "url": "https://boltbait.com/pdn/BoltBaitPack51.zip",
+    "hash": "7d935f3d3edc80f9422d6673317ab7c51548472b493e1826bfeae8bcc9e6f4dd",
+    "pre_install": [
+        "$installerAssembly = [System.Reflection.Assembly]::Load([IO.File]::ReadAllBytes(\"$dir\\BoltBaitPackInstall50.exe\"))",
+        "Remove-Item \"$dir\\BoltBaitPackInstall50.exe\"",
```

Comment:
```suggestion
        "$installerAssembly = [System.Reflection.Assembly]::Load([IO.File]::ReadAllBytes(\"$dir\\BoltBaitPackInstall51.exe\"))",
        "Remove-Item \"$dir\\BoltBaitPackInstall51.exe\"",
```

Reply:
This will change with each version. I recommend first obtaining the name using `Get-Item "$dir\BoltBaitPackInstall*.exe"` and storing it in a variable.

Reply:
Oops. Didn't notice this one. Thanks.